lib/block_view_message_test.go

```diff
@@ -2014,8 +2107,28 @@ func TestGroupMessages(t *testing.T) {
 		// sender -> recipient
 		// 	sender: 4
 		//	recipient: 4
+		// Since we're passed the ExtraData migration, the entry will have the extra data field. We add it after
+		// transaction is processed as an extra sanity-check.
+		messageEntry.ExtraData = make(map[string][]byte)
+		messageEntry.ExtraData[MessagesVersionString] = UintToBuf(MessagesVersion3)
+		messageEntry.ExtraData[SenderMessagingPublicKey] = entry.MessagingPublicKey[:]
+		messageEntry.ExtraData[SenderMessagingGroupKeyName] = NewGroupKeyName(defaultKey).ToBytes()
+		messageEntry.ExtraData[RecipientMessagingPublicKey] = entryRecipient.MessagingPublicKey.ToBytes()
+		messageEntry.ExtraData[RecipientMessagingGroupKeyName] = NewGroupKeyName(defaultKey).ToBytes()
```

Comment:
We keep repeating this chunk of code. Can we move it to a helper function?

Reply:
makes sense